src/components/CharacterCard.js

```diff
@@ -1,5 +1,34 @@
 import React from "react";
+import styled from "styled-components";
 
-export default function CharacterCard() {
-  return <span>todo: character</span>;
+const StyledCard = styled.div`
+  border: 1px solid darksalmon;
+  box-shadow: 0px 0px 22px -1px rgba(87, 81, 87, 0.65);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 350px;
+  margin: 20px;
+  padding: 10px;
+  border-radius: 15px;
+  &&:hover {
+    box-shadow: 10px 5px 40px -1px rgba(87, 81, 87, 0.65);
+  }
+`;
+
+export default function CharacterCard(props) {
+  const { name, status, species, image, gender, origin } = props;
```

Comment:
This is great, you can also but this deconstructed obj in the parameter portion of the function.